dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/CrossVersionReporter.scala

```diff
@@ -0,0 +1,23 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.reporters.Reporter
+
+/**
+ * In update 2.12.12 -> 2.12.13 reporter was changed from method to value.
+ * This code creates a reporter, regardless whether is a value of method
+ */
+object CrossVersionReporter {
```

Comment:
Nit: any chance of sharing this code b/w checker-plugin and dump-event-schema-compiler-plugin?

Reply:
Ofc can be done on #48 or another PR; and ofc refrain from doing so if it requires some non-trivial engineering, code duplication isn't the worst thing that can happen ;)

Reply:
This would either require an entire separate project with this one file.
Making one plugin depend on another is a very bad idea, because compiler plugins are fat jars.

Reply:
Okay, I was afraid of that :/ let's leave as is then